apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/EditSlippage/EditSlippageScreen/EditSlippageScreen.tsx

```diff
@@ -212,6 +212,7 @@ const useStyles = () => {
 
   const colors = {
     background: color.gray_c100,
+    text: color.gray_c900,
```

Comment:
@banklesss can u attest if is it correct, cuz now there are sub-tokens, for all texts -> `text_`

Reply:
thanks 👍